oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/Layer.java

```diff
@@ -126,7 +126,12 @@ public enum Layer {
     /**
      * Cache conjectured by client side plugin(eg. skywalking-java -> JedisPlugin LettucePlugin)
      */
-    VIRTUAL_CACHE(19, false);
+    VIRTUAL_CACHE(19, false),
+
+    /**
+     * PostgreSQL is a powerful, open source object-relational database system
```

Comment:
Don't put a subjective word(powerful) in the comment when refer to 3rd party project.
Such as in this case, ppl could think we only accept monitoring something powerful as a layer. This is not our meaning. 😁

Reply:
I just copy from its official website. I don't know how to describe it.

Reply:
I like this version, it seems more objective.

> PostgreSQL is an advanced, enterprise-class, and open-source relational database system. 

Found from https://www.postgresqltutorial.com/postgresql-getting-started/what-is-postgresql/

Reply:
Got it.